### PR TITLE
Add file names as comments

### PR DESCRIPTION
**Pull request by makrandgupta**
Add comments to indicate which file the code snippets should go into. Makes it a bit easier to follow along

### CODE REVIEW

File: docs/rtk-query/usage/code-generation.mdx [hunk 1]
```diff
@@ -42,6 +44,8 @@ export const emptySplitApi = createApi({
 Generate a config file (json, js or ts) with contents like
 
 ```ts no-transpile
+// openapi-config.ts
```
Comment (phryneas) [inline docs/rtk-query/usage/code-generation.mdx:47]:
```suggestion
```ts no-transpile title="openapi-config.ts"
```

Comment (phryneas) [reply docs/rtk-query/usage/code-generation.mdx:47]: Hey, great input! Could you please format it like that? Then it should show up like in the [Quick Start](https://redux-toolkit.js.org/tutorials/quick-start) code examples (look for `index.js`)

Comment (makrandgupta) [reply docs/rtk-query/usage/code-generation.mdx:47]: ah fantastic, will do!

**Comment by phryneas** [comment]
Looks great, thank you very much!

**Pull request merged**